apis/tempo/v1alpha1/tempostack_types.go

```diff
@@ -655,6 +664,13 @@ type IngressSpec struct {
 	// +kubebuilder:validation:Optional
 	// +operator-sdk:csv:customresourcedefinitions:type=spec,displayName="Route Configuration"
 	Route RouteSpec `json:"route,omitempty"`
+
+	// Security defines the options for the ingress security in single tenant mode.
+	//
+	// +optional
+	// +kubebuilder:validation:Optional
+	// +operator-sdk:csv:customresourcedefinitions:type=spec,displayName="Ingress Security Configuration"
+	Security IngressSecuritySpec `json:"security,omitempty"`
```

Comment:
is the ingress used for both - ingestion and query? Would it make sense to scope the `security` (I don't like the term) for just the jaeger query?

Reply:
Oh you're right, I'll move this to jaegerUI struct. I also accept other suggestions different from security for the name :D

Reply:
Moved!